cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -35,6 +34,13 @@
 ]
 
 
+def remove_vm(provider, vm_name):
+    if provider.mgmt.does_vm_exist(vm_name):
+        if provider.mgmt.is_vm_running(vm_name):
+            provider.mgmt.stop_vm(vm_name)
+        provider.mgmt.delete_vm(vm_name)
```

Comment:
Still, I'm aware we need to have this somewhere in one place, and I think the best place would be in class RHEVMSystem in mgmtsystem/rhevm.py in wrapanapi, because it wouldn't break anything for other providers, but this PR is the fastest way for me to make tests work correctly while I work on a better fix. For now we can just checkout directly from PR in jenkins, with no intent to get this PR merged in its current state